DfE.FindInformationAcademiesTrusts/Pages/Trusts/_TrustNavigation.cshtml

```diff
@@ -34,7 +34,7 @@
           </a>
         </li>
         <li class="ds_side-navigation__item app-side-navigation-highlight-item">
-          <a asp-page="/Trusts/Academies/Details" asp-route-uid="@Model.TrustSummary.Uid" class="ds_side-navigation__link govuk-body govuk-link govuk-link--no-visited-state @GetCurrentLinkClassIf("Academies details")" data-testid="academies-nav">
+          <a asp-page="/Trusts/Academies/Details" asp-route-uid="@Model.TrustSummary.Uid" class="ds_side-navigation__link govuk-body govuk-link govuk-link--no-visited-state @GetCurrentLinkClassIf("Academies in this trust")" data-testid="academies-nav">
```

Comment:
Without looking into it in too much detail is there functional changes as a result of this string changing, if not resolve and ignore this!